ApplicationLibCode/ReservoirDataModel/RigCaseCellResultsData.cpp

```diff
@@ -2897,6 +2898,20 @@ void RigCaseCellResultsData::computeCompletionTypeForTimeStep( size_t timeStep )
 
     if ( !eclipseCase ) return;
 
+    // If permeabilities are generated by calculations, make sure that generated data is calculated
+    // See RicExportFractureCompletionsImpl::generateCompdatValues()
+    for ( const QString& propertyName : { "PERMX", "PERMY", "PERMZ" } )
+    {
+        for ( auto userCalculation : RimProject::current()->gridCalculationCollection()->calculations() )
+        {
+            QString generatedPropertyName = userCalculation->findLeftHandSide( userCalculation->expression() );
+            if ( generatedPropertyName == propertyName )
+            {
+                userCalculation->calculate();
+            }
+        }
+    }
+
```

Comment:
Maybe check that the eclipse case matches here? No need to compute if calculation is for a different case?